src/AddItems.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { capitalize } from './lib/utility';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
```

Comment:
```suggestion
userToken = 'randomTokenOne'
```

You're not using `setUserToken` which is causing a React warning. Since you're not changing the state of `userToken`, you don't need to use `useState`. You can set the userToken directly to `randomTokenOne` or whatever is in local storage!